libkirk/main.py

```diff
@@ -465,8 +471,8 @@ def run(cmd_args: list = None) -> None:
     if args.json_report and os.path.exists(args.json_report):
         parser.error(f"JSON report file already exists: {args.json_report}")
 
-    if not args.run_suite and not args.run_command:
-        parser.error("--run-suite/--run-command are required")
+    if not args.run_suite and not args.run_command and not args.run_pattern:
+        parser.error("At least one of the run commands is required")
```

Comment:
I liked the previous version where you explicitly stated which options are required.

Reply:
It was becoming quite long, I didn't find any valid alternative